src/PortingAssistant.Client/Program.cs

```diff
@@ -71,16 +93,17 @@ static void Main(string[] args)
 
                     var portingAssistantClient = portingAssistantBuilder.GetPortingAssistant();
                     var reportExporter = portingAssistantBuilder.GetReportExporter();
-                    var solutionSettings = cli.IgnoreProjects != null && cli.IgnoreProjects.Count != 0 ?
-                            new AnalyzerSettings
-                            {
-                                IgnoreProjects = cli.IgnoreProjects,
-                                TargetFramework = cli.Target
-                            } : new AnalyzerSettings
-                            {
-                                IgnoreProjects = new List<string>(),
-                                TargetFramework = cli.Target
-                            };
+                    var solutionSettings = cli.IgnoreProjects != null && cli.IgnoreProjects.Count != 0 

```

Comment:
Thanks for the new formatting, looks much better.
Nitpick: My coding habit would be adding parentheses around this (A && B) so it's more straightforward and readable.